bw2io/extractors/ecospold2.py

```diff
@@ -353,6 +487,27 @@ def extract_parameter(cls, exc):
 
     @classmethod
     def extract_properties(cls, exc):
+        """
+        Extract the properties of an exchange.
+
+        Parameters
+        ----------
+        exc : lxml.etree.Element
+            An XML element representing an exchange.
+
+        Returns
+        -------
+        dict
+            A dictionary of the properties of the exchange. Each key in the dictionary
+            is a string representing the name of a property, and the corresponding value
+            is a dictionary with the following keys:
+
+            - "amount" (float): The numerical value of the property.
```

Comment:
Maybe move this to 'Notes' section of the docstring? Check the NumPy standard before.

Reply:
Moved to Notes section.